x-pack/plugins/security_solution/public/common/components/events_viewer/events_viewer.test.tsx

```diff
@@ -337,7 +337,9 @@ describe('EventsViewer', () => {
           <EventsViewer
             {...eventsViewerDefaultProps}
             graphEventId={undefined}
-            headerFilterGroup={<AlertsTableFilterGroup onFilterGroupChanged={jest.fn()} />}
+            headerFilterGroup={
```

Comment:
why are we updating?

Reply:
the headerFilterGroup should not be used anymore

Reply:
the `AlertsTableFilterGroup` component updated, but we should probably delete/modify most of these since we don't use it this way in the app anymore

Reply:
@XavierM we still have the property there though, no? I don't think we need this many tests anymore but it might be worthwhile to keep some around in case we use it with the component somewhere down the line.